frontend/components/wallet/WalletProvider.tsx

```diff
@@ -116,10 +116,11 @@ export const WalletProvider: React.FC<Props> = ({ children }) => {
           siweMessage,
           signature as string
         );
+        console.log("The account's Main Controller address is:", signerAddress);
```

Comment:
here too

Reply:
@democrazyfi specifically left it in so we can monitor behavior for a bit